src/actions/dataset.ts

```diff
@@ -44,6 +45,7 @@ export function datasetUrlLoad(name: string, url: string): DatasetUrlLoad {
           type: DATASET_URL_RECEIVE,
           payload: {name, url, schema}
         });
+        return { schema };
```

Comment:
we no longer have to. we can remove. it was for chaining promises that isn't currently used.